jira/client.py

```diff
@@ -2910,26 +2892,24 @@ def search_issues(
 
         Args:
             jql_str (str): The JQL search string.
-            startAt (int): Index of the first issue to return. (Default: 0)
-            maxResults (int): Maximum number of issues to return. Total number of results
-              is available in the ``total`` attribute of the returned :class:`~jira.client.ResultList`.
-              If maxResults evaluates as False, it will try to get all issues in batches. (Default: 50)
-            validate_query (bool): Whether or not the query should be validated. (Default: True)
+            startAt (int): Index of the first issue to return. (Default: ``0``)
+            maxResults (int): Maximum number of issues to return.
+              Total number of results is available in the ``total`` attribute of the returned :class:`ResultList`.
+              If maxResults evaluates to False, it will try to get all issues in batches. (Default: ``50``)
+            validate_query (bool): True to validate the query. (Default: ``True``)
             fields (Optional[Union[str, List[str]]]): comma-separated string or list of issue fields to include in the results.
               Default is to include all fields.
             expand (Optional[str]): extra information to fetch inside each resource
-            json_result (bool): JSON response will be returned when this parameter is set to True.
-              Otherwise, :class:`~jira.client.ResultList` will be returned.
+            json_result (bool): True to return a JSON response. When set to False a :class:`ResultList` will be returned. (Default: ``False``)
 
         Returns:
             Union[Dict,ResultList]: Dict if ``json_result=True``
-
         """
         if isinstance(fields, str):
             fields = fields.split(",")
 
-        # this will translate JQL field names to REST API Name
-        # most people do know the JQL names so this will help them use the API easier
+        # this will translate JQL field names to REST API Name most people do know the
+        # JQL names so this will help them use the API easier
```

Comment:
I would prefer the original in this case

the first line reads like a summary "this will translate JQL field names to REST API Name"
The second line is then treated more as a way to add more details on why.

By combining them together like this it makes it harder to see that